temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowOutboundCallsInterceptor.java

```diff
@@ -87,11 +217,14 @@ public <R> LocalActivityOutput<R> executeLocalActivity(LocalActivityInput<R> inp
   @Override
   public <R> ChildWorkflowOutput<R> executeChildWorkflow(ChildWorkflowInput<R> input) {
     if (!WorkflowUnsafe.isReplaying()) {
+      Span capturedSpan = tracer.scopeManager().activeSpan();
       Span childWorkflowStartSpan =
           contextAccessor.writeSpanContextToHeader(
               () -> createChildWorkflowStartSpanBuilder(input).start(), input.getHeader(), tracer);
       try (Scope ignored = tracer.scopeManager().activate(childWorkflowStartSpan)) {
-        return super.executeChildWorkflow(input);
+        ChildWorkflowOutput<R> output = super.executeChildWorkflow(input);
+        return new ChildWorkflowOutput<>(
+            new PromiseWrapper<>(capturedSpan, output.getResult()), output.getWorkflowExecution());
```

Comment:
`output.getWorkflowExecution()` is also a Promise that needs to be wrapped

Reply:
Good catch.  Will fix.

Reply:
fixed